packages/docs/verite/patterns/identifier.md

```diff
@@ -26,3 +24,13 @@ While implementors may choose any method they like, some common factors include:
 - the ability for issuers, verifiers, and holder wallets to resolve DIDs without relying on a specific tenant or chain
 
 In general, DID method options will be influenced by a broader set of criteria relevant to all roles in the VC ecosystem, a comprehensive consideration of which is available in the [DID Method Rubric](https://w3c.github.io/did-rubric).
+
+## Wallet-based versus Address-based Holder Identification Schemes
```

Comment:
```suggestion
## Wallet-based versus Account-based Holder Identification Schemes
```

Reply:
Not sure about my suggestion. Would this replacement allow for more consistency with the other docs that refer to "account-based"? Ignore this if I'm missing something